### DISCUSSION TITLE
Folder Templates not showing/disappeared

### DISCUSSION THREAD

**Discussion by Lobobp** [SilentVoid13/Templater#1175] [Help]
Hi, I want to set up some folder templates but the whole section is not there.  Like, it's supposed to be between "Template Hotkeys" and "Startup Templates", but it doesnt show anything. Im not sure if it's because of some incompatibility with other plugin. Pasted an image so you can see. Also an image of the plugins Im using, in case this is a known thing.

![image](https://github.com/SilentVoid13/Templater/assets/143235896/e874ff57-f601-40a2-a907-f343c94f336b)
![image](https://github.com/SilentVoid13/Templater/assets/143235896/fd301e32-2f7b-4d4f-adca-b01a2fc95359)

**Comment by Zachatoo** [comment]
You have to turn on "Trigger Template on new file creation" first

> Note: This setting is hidden by default. To view it first enable the Trigger Template on new file creation setting.

https://silentvoid13.github.io/Templater/settings.html#folder-templates

**Reply by Lobobp** [reply]
Oh, yeah that worked. Thanks! 😄
